docs/advanced/AsyncActions.md

```diff
@@ -390,7 +390,7 @@ How do we include the Redux Thunk middleware in the dispatch mechanism? We use t
 
 ```js
 import thunkMiddleware from 'redux-thunk'
-import createLogger from 'redux-logger'
+import {createLogger} from 'redux-logger'
```

Comment:
Can you add spaces inside of the brackets? That matches the style of the rest of the document.